src/mgr/PyModules.cc

```diff
@@ -29,6 +29,34 @@
 
 #define dout_context g_ceph_context
 #define dout_subsys ceph_subsys_mgr
+
+#undef dout_prefix
+#define dout_prefix *_dout << "mgr [py] "
```

Comment:
I'd suggest this be "mgr[$module_name]" if possible, rather than "mgr [py]", so you know which module the log/error came from.